sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala

```diff
@@ -683,7 +683,7 @@ abstract class SparkStrategies extends QueryPlanner[SparkPlan] {
         execution.LocalLimitExec(limit, planLater(child)) :: Nil
       case logical.GlobalLimit(IntegerLiteral(limit), child) =>
         execution.GlobalLimitExec(limit, planLater(child)) :: Nil
-      case logical.Union(unionChildren) =>
+      case logical.Union(unionChildren, _, _) =>
```

Comment:
this pattern appears many times (`Union(children, _, _)`), and actually we can rewrite to
```
case union: logical.Union =>
  execution.UnionExec(union.children.map(planLater)) :: Nil
```

Reply:
oh ok.